### PR TITLE
rename folder, add annotation to fake policy data

### PR DESCRIPTION
**Pull request by sailorlqh**
fixed:#101

### CODE REVIEW

File: test/e2e/testdata/fakedata/mapping/product_team_a/product_x/x_mapping.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ resource:
   name: '//storage.googleapis.com/pmap-dev-manual-bucket'
   provider: 'gcp'
 annotations:
-  traceID: 'fake-bets-pmap-dev-manual-test'
+  traceID: 'fake-bets-pmap-dev-manual-test-mapping'
```
Comment (capri-xiyue) [inline test/e2e/testdata/fakedata/mapping/product_team_a/product_x/x_mapping.yaml:5]: Can you also get rid of `bets` here?

File: test/e2e/testdata/fakedata/policy/product_team_b/product_y/y_policy.yaml [hunk 2]
```diff
@@ -1,5 +1,7 @@
 resource:
   name: 'policy'
+annotations:
+  traceID: 'fake-bets-pmap-dev-manual-test-policy'
```
Comment (capri-xiyue) [inline test/e2e/testdata/fakedata/policy/product_team_b/product_y/y_policy.yaml:4]: Same here, get rid of `bets`

**Comment by capri-xiyue** [comment]
Can you update the description like `fixed #101`

File: test/e2e/testdata/fakedata/mapping/product_team_a/product_x/x_mapping.yaml (hunk 1 above)
Comment (sailorlqh) [reply test/e2e/testdata/fakedata/mapping/product_team_a/product_x/x_mapping.yaml]: Done, I also removed `bets` from the workflow name, and yml file name.

**Pull request merged**